sphinxgallery/gen_rst.py

```diff
@@ -310,81 +339,57 @@ def execute_script(image_dir, thumb_file, image_fname, base_image_name,
     if os.path.exists(time_path):
         time_elapsed = float(open(time_path).read())
 
-    if not os.path.exists(first_image_file) or \
-       os.stat(first_image_file).st_mtime <= os.stat(src_file).st_mtime:
-        # We need to execute the code
-        print('plotting %s' % fname)
-        t0 = time()
-        import matplotlib.pyplot as plt
```

Comment:
Its purpose would be to not require matplotlib to compile documentation
that used the gallery without plotting...

On 8 June 2015 at 18:26, Loïc Estève <notifications@github.com> wrote:

> In sphinxgallery/gen_rst.py
> <https://github.com/sphinx-gallery/sphinx-gallery/pull/36#discussion_r31894523>
> :
>
> > @@ -310,81 +339,57 @@ def execute_script(image_dir, thumb_file, image_fname, base_image_name,
> >      if os.path.exists(time_path):
> >          time_elapsed = float(open(time_path).read())
> >
> > -    if not os.path.exists(first_image_file) or \
> > -       os.stat(first_image_file).st_mtime <= os.stat(src_file).st_mtime:
> > -        # We need to execute the code
> > -        print('plotting %s' % fname)
> > -        t0 = time()
> > -        import matplotlib.pyplot as plt
>
> Makes sense, I misread the diff.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sphinx-gallery/sphinx-gallery/pull/36/files#r31894523>
> .
>